cvmfs/cvmfs.cc

```diff
@@ -1829,123 +1830,128 @@ static int Init(const loader::LoaderExports *loader_exports) {
   cvmfs::backoff_throttle_ = new BackoffThrottle();
 
   // Option parsing
-  options::Init();
+  if (cvmfs::loader_exports_->simple_options_parsing) {

```

Comment:
Older versions of the loader won't have the simple_options_parsing field. Please add the logic to prepare for that (see comment below regarding versioning).